charts/careers/templates/web_deployment.yaml

```diff
@@ -37,18 +37,17 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          command: ["python", "manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
yeah, which is now a wsgi command instead of this dev runserver, but the contract ought to be the same I believe, though perhaps the port is wrong (I'm debugging why tests are failing for this ci)